workspaces/ui/src/contexts/CaptureContext.js

```diff
@@ -51,6 +51,16 @@ class _CaptureContextStore extends React.Component {
     );
   };
 
+  componentDidUpdate(prevProps, prevState, snapshot) {
```

Comment:
how often does this run, is it performance intensive, and is there a better way?

Reply:
I think at some point during the demo implementation that pathway to refresh the diff after finalize was circumvented. I'm able to replicate this issue from Victor on every new endpoint. It only runs once. 

Audited the code to make sure. Start diff only happens on....
- componentDidMount
- new commands applied
- whenever the outer rfc state changes

<img width="974" alt="Screen Shot 2020-09-08 at 2 27 03 PM" src="https://user-images.githubusercontent.com/5900338/92514143-70c23c00-f1df-11ea-99f0-6cf76a8dc3b0.png">